cloudflare/resource_cloudflare_zone_settings_override.go

```diff
@@ -467,6 +475,13 @@ var resourceCloudflareZoneSettingsSchema = map[string]*schema.Schema{
 		Optional:     true,
 		Computed:     true,
 	},
+
+	"0rtt": {
```

Comment:
Aren't attributes supposed to be identifiers that don't start with a number?

I may not be on the version that this was added. But I get an `An argument or block definition is required here.` error at the start of `0rtt = "off"` when including it in an cloudflare_zone_settings_override.

Reply:
I'll have to have a look. The alpha characters restrictions for other things are associated with resource naming/use but I haven't dug into whether that applies for the schema.

Reply:
It does.  This attribute needs to be renamed to something like `zero_rtt` instead.  See the [HCL scanner code](https://github.com/hashicorp/hcl/blob/master/hcl/scanner/scanner.go#L165-L166) for details.